main/views.py

```diff
@@ -80,68 +80,49 @@ class ProcessAction(Enum):
     FLUSH = "flush"
 
 
-async def _process_call(uuid: str, action: ProcessAction) -> None:
+async def _process_call(uuids: list[str], action: ProcessAction) -> None:
     """Perform an action on a process with a given UUID.
 
     Args:
-        uuid: UUID of the process to be actioned.
-        action: Action to be performed {restart,kill}.
+        uuids: List of UUIDs of the process to be actioned.
+        action: Action to be performed {restart,flush,kill}.
     """
     pmd = get_process_manager_driver()
-    query = ProcessQuery(uuids=[ProcessUUID(uuid=uuid)])
+    uuids_ = [ProcessUUID(uuid=u) for u in uuids]
 
     match action:
         case ProcessAction.RESTART:
-            await pmd.restart(query)
+            for uuid_ in uuids_:
+                query = ProcessQuery(uuids=[uuid_])
+                await pmd.restart(query)
```

Comment:
Why do we need this different logic here? It makes things rather awkward and I'd be tempted to stick with the previous implementation, particularly if we take out the awaits.

Reply:
`pmd.restart` do not work with more than one `uuid` at the same time, so this one needs a loop. Where we put the loop, I don't mind.